ibraheemosule/skoolizy
# Added Route name using the useLocation hook.

# Why this PR?
To let users know the current page/route they have navigated to.

# What does this PR do?
It displays the current page route to the user when navigating and also a capitalize function was introduced to capitalize the first letter of every string.

# How to test your work?
Navigate through different pages, you should see the route names for the different pages displayed and also the first letter of the route should be capitalized.

# Pages to see your changes?
- TopHeader.tsx
